src/server/svr_func.c

```diff
@@ -6865,3 +6866,78 @@ memory_debug_log(struct work_task *ptask)
 	}
 #endif /* malloc_info */
 }
+
+/**
+ * @brief
+ *		Get list of deferred requests for a particular scheduler.
+ * 		If the list does not exist yet and the 'create' is TRUE,
+ * 		then create the list.
```

Comment:
Maybe rename this function?  Calling a function called "get" and having it create something seems confusing to me.

Reply:
I think "fetch" is more like "get or create". Changed.